keras/backend/mxnet_backend.py

```diff
@@ -3157,7 +3157,39 @@ def conv3d_transpose(x, kernel, output_shape, strides=(1, 1, 1),
     # Raises
         ValueError: if `data_format` is neither `channels_last` or `channels_first`.
     """
-    raise NotImplementedError('MXNet Backend: conv3d_transpose operator is not supported yet.')
+    if data_format is None:
+        data_format = image_data_format()
+    _validate_data_format(data_format)
+
+    if padding not in {'same', 'valid'}:
+        raise ValueError('`padding` should be either `same` or `valid`.')
+
+    # Handle Data Format
+    x = _preprocess_convnd_input(x, data_format)
+    kernel = _preprocess_convnd_kernel(kernel, data_format)
+
+    # We have already converted kernel to match MXNet required shape:
+    # (depth, input_depth, rows, cols)
+    kernel_shape = kernel.shape
```

Comment:
These code are common across conv2d_transpose and conv3d_transpose operator. Please refactor and avoid code duplication.